pyTigerGraph/gds/dataloaders.py

```diff
@@ -1621,20 +1622,29 @@ def __init__(
             self._etypes = list(self._e_schema.keys())
         self._vtypes = sorted(self._vtypes)
         self._etypes = sorted(self._etypes)
+        if v_seed_types:
+            if isinstance(v_seed_types, list):
+                self._seed_types = v_seed_types
+            elif isinstance(v_seed_types, str):
+                self._seed_types = v_seed_types
```

Comment:
Should this be different from line 1627?

Reply:
Whoops, should be wrapped in a list.